zhmc_prometheus_exporter/zhmc_prometheus_exporter.py

```diff
@@ -2407,7 +2421,26 @@ def main():
                                    hmc_info['api-minor-version'])
                 client = zhmcclient.Client(session)
                 hmc_features = client.consoles.console.list_api_features()
+
+                # Determine list of CPCs to export, as specified in 'cpcs'.
+                # 'cpcs' is optional and defaults to all managed CPCs.
                 cpc_list = client.cpcs.list()
+                cpc_names = [cpc.name for cpc in cpc_list]
```

Comment:
Is cpc_names actually read anywhere?

Reply:
Yes, this is used:
* in the loop that determines the non-existing CPC names specified in the config file (`ne_cpc_names`)
* if the config file does not specify `cpcs`, it is not replaced and is used further down in the `logprint()` call.